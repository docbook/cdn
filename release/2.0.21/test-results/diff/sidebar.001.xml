<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">This is a sidebar test.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">sidebar</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
            <header deltaxml:deltaV2="A" class="sidebar-titlepage">
               <h3>Sidebar</h3>
            </header><p deltaxml:deltaV2="B">This is a sidebar test.</p>
            <p deltaxml:deltaV2="B">Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="B">This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">sidebar-content</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">sidebar</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
               <p deltaxml:deltaV2="A">Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><header deltaxml:deltaV2="B" class="sidebar-titlepage">
                  <h3>Sidebar</h3>
               </header>
               <p deltaxml:deltaV2="A">This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><div deltaxml:deltaV2="B" class="sidebar-content">
                  <p>Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p>
                  <p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p>
               </div>
            </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="B">Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>