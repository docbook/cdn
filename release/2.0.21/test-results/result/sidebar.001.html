<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for sidebar.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: sidebar.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">13</span> differences)</th><th>Actual (<span class="adiff">11</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;This is a sidebar test.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">sidebar</span>"</span>&gt;
            <span class="tag diff del">&lt;header <span class="att">class="sidebar-titlepage"</span>&gt;
               <span class="tag">&lt;h3&gt;Sidebar&lt;/h3&gt;</span>
            &lt;/header&gt;</span>
            <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">sidebar-content</span>"</span>&gt;
               <span class="tag diff del">&lt;p&gt;Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span>
               <span class="tag diff del">&lt;p&gt;This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
            <span class="tag diff add">&lt;p&gt;This is a sidebar test.&lt;/p&gt;</span>
            <span class="tag diff add">&lt;p&gt;Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;p&gt;This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span><span class="diff txtchg">
            </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">sidebar</span>"</span>&gt;
               <span class="tag diff add">&lt;header <span class="att">class="sidebar-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;Sidebar&lt;/h3&gt;</span>
               &lt;/header&gt;</span>
               <span class="tag diff add">&lt;div <span class="att">class="sidebar-content"</span>&gt;
                  <span class="tag">&lt;p&gt;Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span>
                  <span class="tag">&lt;p&gt;This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;p&gt;Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
&lt;/p&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This is a sidebar test.</p><p>Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><div class="sidebar"><header class="sidebar-titlepage"><h3>Sidebar</h3></header><div class="sidebar-content"><p>Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p></div></div><p>Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><p>This is a sidebar test.</p><p>Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><div class="sidebar"><header class="sidebar-titlepage"><h3>Sidebar</h3></header><div class="sidebar-content"><p>Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p></div></div><p>Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/sidebar.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>