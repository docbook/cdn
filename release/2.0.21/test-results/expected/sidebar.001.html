<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>This is a sidebar test.</p><p>Here is some content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><div class="sidebar"><header class="sidebar-titlepage"><h3>Sidebar</h3></header><div class="sidebar-content"><p>Here is some sidebar content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p><p>This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p></div></div><p>Back into the main flow. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
This is content. This is content. This is content. This is content.
</p></article><script src="../../resources/js/prism.js"></script></body></html>